### ISSUE TITLE
Starter Example

### ISSUE THREAD

**Issue by marcusjwhelan** [actix/actix-web#932]
Cannot get the starter project working. Current code

```rust
extern crate actix_web;
use actix_web::{web, App, HttpServer, Responder};

fn index(info: web::Path<(String, u32)>) -> impl Responder {
    format!("Hello {}! id:{}", info.0, info.1)
}

fn main() -> std::io::Result<()> {
    HttpServer::new(|| App::new().service(
        web::resource("/{name}/{id}/index.html").to(index))
    )
        .bind("127.0.0.1:8080")?
        .run()
}
```

however I get this error 

> The character encoding of the plain text document was not declared. The document will render with garbled text in some browser configurations if the document contains characters from outside the US-ASCII range. The character encoding of the file needs to be declared in the transfer protocol or file needs to use a byte order mark as an encoding signature.

Firefox browser

**Comment by HitomiTenshi** [comment]
What's the `content-type` response header you get when calling e. g. http://127.0.0.1:8080/Bane/1/index.html?

It should be: `content-type: text/plain; charset=utf-8`

Check the network tab in your browser's dev tools.

If the charset part is missing you might have an outdated `actix-web` dependency. Try running `cargo update` in your project root folder and run the example again. If it's there but the error keeps happening, check if you're running the latest version of Firefox.

Edit: You can also try updating your Rust toolchain by running `rustup update`. `actix-web` currently supports a minimum Rust version of `1.34` or later.

**Comment by fafhrd91** [comment]
it is not a problem of backend code, use `curl` instead of browser

interaction with browser is outside of scope

**Comment by marcusjwhelan** [comment]
@HitomiTenshi Ohhhh. I see, I guess I have never seen starter code that just didn't work without reading it haha. Didn't realize the url needed an input. I am completely new to this back-end and was unsure how anything worked so I did not know that meant I needed to supply parameters. @fafhrd91 maybe that info could be added to the docs?

Also @fafhrd91, what is the proper curl command as well? It would be nice to have an example of that as well.
